Library/Homebrew/cask/url.rb

```diff
@@ -99,7 +99,7 @@ def initialize(
 
     class BlockDSL
       # Allow accessing the URL associated with page contents.
-      module PageWithURL
+      class PageWithURL < String
```

Comment:
I've had bad experiences subclassing the core library, as have [others](https://steveklabnik.com/writing/beware-subclassing-ruby-core-classes/). A more common rubyism is to subclass [`SimpleDelegator`](https://docs.ruby-lang.org/en/3.3/SimpleDelegator.html), then initiate it with the string you want to delegate to.

Reply:
Looks great! I wasn't sure how sorbet-compatible it would be, but you were able to make it work without any rbi shenanigans. 🅰️➕ work!